static/src/javascripts/projects/common/modules/atoms/youtube-player.ts

```diff
@@ -165,7 +165,10 @@ const createAdsConfigDisabled = (): AdsConfigDisabled => {
 const createAdsConfigEnabled = (
 	consentState: ConsentState,
 ): AdsConfigEnabled => {
-	const custParams = getPageTargeting() as Record<string, MaybeArray<string>>;
+	const custParams = getPageTargeting(consentState) as Record<
+		string,
+		MaybeArray<string>
+	>;
```

Comment:
Can we get rid of the type assertion, now?

Reply:
This is a little tricky as `custParams` is passed to the function `constructQuery` which is expecting defined types whereas `getPageTargeting` is returning `PartialWithNulls`.

I'm going to shift this to another PR as I think we need to revisit the `type PageTargeting = PartialWithNulls<{...}>` type. The return type of `getPageTargeting` shouldn't return any properties with falsy values so I think its safe to have just a partial.

I think that would then allow the consumers to have narrower types.